pages/cards/overview/big-arcana/index.js

```diff
@@ -0,0 +1,36 @@
+import Link from "next/link";
+import { useState, useEffect } from "react";
+import BackButton from "../../../../components/Backbutton/backbutton";
+import useStore from "../../../../src/store/store";
+
+export default function Overview() {
+  const [hasMounted, setHasMounted] = useState(false);
+
+  const cards = useStore((state) => state.allCards);
+
+  useEffect(() => {
+    setHasMounted(true);
+  }, []);
+  if (!hasMounted) {
+    return null;
+  }
+  return (
+    <>
+      <ul>
+        <h2>big arcana</h2>
```

Comment:
The direct child of an `ul` element should be a `li`, so please move the `h2` above the list.